tutorials-v5/heom/heom-6-parity.md

```diff
@@ -0,0 +1,244 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.16.0
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+```python
+import numpy as np
+import matplotlib.pyplot as plt
+from qutip import tensor,destroy,qeye,spre,spost,operator_to_vector,expect,qeye
+from qutip.solver.heom import FermionicBath,LorentzianPadeBath,HEOMSolver
+import scipy.sparse as sp
+from scipy.sparse.linalg import lgmres
+```
+
+<!-- #region -->
+# Kondo  Peak
+
+In this tutorial we will find the kondo peak for a system of two spins connected to two fermionic reservoirs 
+
+The Hamiltonian of this setup is given by:
+
+$H_{T}=H_{S}+H_{f}+H_{ef}$
+
+Where $H_{S}$ is the system Hamiltonian, which is divided into three contributions. The electron system hamiltonian ($H_{e}$), the single-mode cavity ($H_{c}$) and the light matter coupling ($H_{ef}$):
+
+$H_{S} = H_{e} + H_{ec}$
+
+
+$H_{e}=  \sum_{n=g,e}\sum_{\sigma=\uparrow,\downarrow} \epsilon_{n} \hat{n}^{n \sigma} + U_{n} \hat{n}_{n \uparrow} \hat{n}_{n \downarrow} $
+
+$H_{ec} = \sum_{\sigma=\uparrow,\downarrow} g_{ct} (d_{g\sigma}^{\dagger} d_{e\sigma} + d_{e\sigma}^{\dagger} d_{g\sigma} )(a^{\dagger}+a)$
+
+
+The other two terms in the total Hamiltonian are given by
+
+$H_{f}= \sum_{\alpha} \sum_{k} \epsilon_{\alpha,k} c^{\dagger}_{\alpha,k}c_{\alpha,k}$
+
+and
+
+$H_{ef}= \sum_{k} \sum_{\alpha=L,R} \sum_{\sigma=\uparrow,\downarrow} g_{\alpha,k} (c_{\alpha,k}^{\dagger} d_{g\sigma}+c_{\alpha,k} d_{g\sigma}^{\dagger})$
```

Comment:
To simplify things i would remove the left/right label here. While one can derive this model from a two-lead one, in practise below we only one ''lead'' split up into two spin labelled baths.